test/deprecated/old-eigvals.chpl

```diff
@@ -0,0 +1,106 @@
+use LinearAlgebra, Random;
+
+config const epsilon = 0.0000000001;
+
+proc isClose(a: complex, b: complex) {
```

Comment:
Should this do a relative comparison like in [LCALS](https://github.com/chapel-lang/chapel/blob/master/test/release/examples/benchmarks/lcals/LCALSChecksums.chpl#L60)?

Reply:
Maybe so. Note this is just a copy of `testEigen.chpl` before it was modified in this PR, so we'd want to fix it in both places.